### ISSUE TITLE
Cannot move the cards into a line

### ISSUE THREAD

**Issue by paigeheff** [paigeheff/responsive-icon-cards#1]
I am trying to get the cards to move into a line on the top (like the screenshot), but it doesn't seem to be working (display inline block isnt working and width isn't working). To space out the cards like in screenshot 3 and 4, would I just change the width ? 
 It says on markbot that I cannot use float (I used that for the image before to go to the left), so how would I make the img go to the left? 
also, I cannot seem  to center the circle img to the text. (I tried transform, vertical-align etc). 

My font also looks alot bolder than your font.

@thomasjbradley

**Comment by thomasjbradley** [comment]
The biggest thing I can see right now on the website is that each icon card isn't grouped together. Each one needs to be in its own element so the icon and text can move as a single object.

**Comment by paigeheff** [comment]
I put the img and p into their own div's. But doesn't the image move separately from the text? Because sometimes the image is on the left and sometimes it is in the middle of the text. 

I am a bit confused about this one.

**Comment by thomasjbradley** [comment]
That's looking better—what about putting more `<div>` tags:
- Another to surround just the `<img>`
- And another to surround just heading & paragraph

Both inside the `<div>` that you already have.

**Comment by paigeheff** [comment]
yes ! it worked! the image is now centered! It is also saying I need the html in the @media... but what do I put into the html? Isn't the html the whole entire thing?

**Comment by thomasjbradley** [comment]
These things are needed in `html` to make the font-size/line-height adjust on different screen sizes:
https://learn-the-web.algonquindesign.ca/topics/screen-sizes-cheat-sheet/
